cmd/init/config.go

```diff
@@ -34,18 +39,37 @@ func generatehcl2NixConf(pt langdetect.ProjectType, pd *langdetect.ProjectDetail
 		}
 		return config, nil
 	case langdetect.BaseImage:
-		return generateEmptyConf(baseImgName), nil
+		return generateEmptyConf(baseImgName, addCommonDeps), nil
 	default:
 		return hcl2nix.Config{
 			Packages: hcl2nix.Packages{},
 		}, fmt.Errorf("language is not supported")
 	}
 }
 
-func generateEmptyConf(imageName string) hcl2nix.Config {
+func generateEmptyConf(imageName string, addCommonDeps bool) hcl2nix.Config {
+	pt, _, err := langdetect.FindProjectType()
+	if err != nil {
+		fmt.Println(styles.ErrorStyle.Render("error:", err.Error()))
+		os.Exit(1)
+	}
```

Comment:
We cant detect what type of project it is if it is a base image as it will always give `langdetect.BaseImage`. We can have another prompt in `init/init.go` as I mentioned yesterday like `Which common deps would you like to add? (Go/Python/Rust)` and then pass it on to here just like we are passing the bool value.

Reply:
It won't always be a project a user might be initialising in, it can be an empty directory as well...

Reply:
When I used langdetect.ProjectType it was giving me the same BaseImg, but when using pt, _, err := langdetect.FindProjectType() it was providing me the "GoModule" for go project and "JsNpm" for Js node project.

I will add "Which common deps would you like to add? " Part and fix it.